nimare/meta/cbma/mkda.py

```diff
@@ -195,6 +196,7 @@ def __init__(self, kernel_transformer=MKDAKernel, prior=0.5, **kwargs):
 
         self.prior = prior
 
+    @use_memmap(LGR)
```

Comment:
@use_memmap(LGR, n_files=2) (but I think this decorator should go in the `PairCBMAEstimator` class)

Reply:
Will it work if the decorator is applied to the whole class or `fit()` instead of `_fit()`?

Reply:
in `fit` instead of `_fit`, sorry that was not clear

Reply:
After digging into the CorrelationDecoder error a bit, it seems like the decorator impacts `inspect.getfullargspec`:

```python
import inspect
from nimare.meta.cbma import MKDAChi2, MKDADensity

print(inspect.getfullargspec(MKDAChi2.fit).args)
print(inspect.getfullargspec(MKDAChi2._fit).args)
print(inspect.getfullargspec(MKDADensity.fit).args)
print(inspect.getfullargspec(MKDADensity._fit).args)
```

I would expect to get:

```
['self', 'dataset1', 'dataset2']
['self', 'dataset1', 'dataset2']
['self', 'dataset']
['self', 'dataset']
```

However what I get is:

```
['self']
['self', 'dataset1', 'dataset2']
['self', 'dataset']
['self']
```

Since the decorator is applied to `fit` for `MKDAChi2` and `_fit` for `MKDADensity`, it looks like those are the ones that are affected.

Reply:
I actually need `n_files` to be 3 in ALESubtraction because the difference scores can be mem-mapped, so I think it's probably best to switch back to decorating `_fit`. Maybe I could decorate a new abstractmethod within PairwiseCBMAEstimator?